### DISCUSSION TITLE
Update from 10.10 to 10.11. NativeMenu on OSX

### DISCUSSION THREAD

**Discussion by Megapit** [AvaloniaUI/Avalonia#7332] [Q&A] [answered]
Hi

I just updated my little program to 10.11. After that i realize that my NativeMenu in app.xaml has changed its behavior.
In 10.10 my own menuitems are shown beside other entrys from osx linke quit, about and so on.
In 10.11 only my menuitems are shown. If i comment out my menuitems in app.xaml the standard osx items are back again.

Is thre a way to bring back my items beside the base osx items in 10.11

**Comment by maxkatz6** [comment]
Do you have disabled setting of MacOSPlatformOptions.DisableDefaultApplicationMenuItems  ?
You can add these items manually, but it should be added automatically unless you disabled it. See:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Native/AvaloniaNativeMenuExporter.cs#L83-L137

**Comment by Megapit** [comment]
Hi

I did not make any changes to it. Only updated to 10.11.
But for test i added the last line in BuildAvaloniaApp in Programm.cs:

public static AppBuilder BuildAvaloniaApp()
          => AppBuilder.Configure<App>()
              .UsePlatformDetect()
              .LogToTrace()              
              .UseReactiveUI()
              .With(new MacOSPlatformOptions { DisableDefaultApplicationMenuItems = false });
  }

But still no changes.
The problem is that i develop in Windows and only run it in OSX. So its a little bit difficult to test :(

**Comment by Megapit** [comment, marked as answer]
Installed the latest nightly. The problem is solved!

[Looks like heavy work is going on in this section:](https://github.com/AvaloniaUI/Avalonia/commits/master/src/Avalonia.Native)
